examples/streamlit-app/requirements.txt

```diff
@@ -0,0 +1,2 @@
+streamlit==1.31.0
+openllmtelemetry[openai]==0.0.1b15
```

Comment:
do we want to version lock? Thoughts on this @jamie256 ?

Reply:
It should be fine to allow >=0.0.1b15

Also suggest using both bedrock and openai extras